ui/widgets/top-bar/src/components/index.tsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import ReactDOM from 'react-dom/client';
```

Comment:
hmm, this won't be captured by webpack for externals extraction

Reply:
good catch, I was just testing...